targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/hal_platform.h

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
We can't accept files with headers like that. You'll need to change it, Apache 2.0 is standard for mbed OS.